benchmarks/cugraph/pytest-based/bench_algos.py

```diff
@@ -111,179 +178,228 @@ def reinitRMM(managed_mem, pool_alloc):
 #
 # For benchmarks, the operations performed in fixtures are not measured as part
 # of the benchmark.
+
 @pytest.fixture(scope="module",
-                params=fixture_params)
-def edgelistCreated(request):
-    """
-    Returns a new edgelist created from a CSV, which is specified as part of
-    the parameterization for this fixture.
-    """
+                params=rmm_fixture_params)
+def rmm_config(request):
     # Since parameterized fixtures do not assign param names to param values,
     # manually call the helper to do so. Ensure the order of the name list
     # passed to it matches if there are >1 params.
     # If the request only contains n params, only the first n names are set.
-    setFixtureParamNames(request, ["dataset", "managed_mem", "pool_allocator"])
-
-    csvFileName = request.param[0]
-    reinitRMM(request.param[1], request.param[2])
-    return utils.read_csv_file(csvFileName)
+    setFixtureParamNames(request, ["managed_mem", "pool_allocator"])
+    reinitRMM(request.param[0], request.param[1])
```

Comment:
I think `reinitializing` RMM pool will fail if you have anything lying around in RMM memory.

Reply:
Agreed. In fact, based on the observations noted [here](https://github.com/rapidsai/cugraph/pull/3540#discussion_r1198366436) and the suggestion [here](https://github.com/rapidsai/cugraph/blob/f4433f499403aa04e0393e9cef21306cbb5497ea/benchmarks/cugraph/pytest-based/conftest.py#L25), I'm wondering if I should change this from being marker-based (which implies it could run multiple combinations of RMM configurations within a single test suite run) to something set only once using a custom CLI option in `conftest.py`.

I'm going to add a FIXME to say that unless you object.